tests/unit_test/admin/admin_processor_test.py

```diff
@@ -45,3 +45,18 @@ def test_get_secrets_empty_value(self):
         BotSecrets(secret_type=BotSecretType.gpt_key.value, value=value, bot=bot, user=user).save()
         secret = Sysadmin.get_bot_secret(bot, BotSecretType.gpt_key.value)
         assert secret == value
+
+    def test_add_bot_secret(self):
+        bot = "test_bot"
+        user = "test_user"
+        secret = "test_secret"
+        bot_secret_id = Sysadmin.add_bot_secret(bot, user, BotSecretType.gpt_key.value, secret)
+        assert bot_secret_id
+        assert BotSecrets.objects(bot=bot, secret_type=BotSecretType.gpt_key.value).get()
```

Comment:
also assert that secret that you added is encrypted by decrypting the value of secret returned from this document